### DISCUSSION TITLE
Is Go-To-Dash feasible? (Can scripts run on blog pages, in addition to the dash?)

### DISCUSSION THREAD

**Discussion by inklesspen** [AprilSylph/XKit-Rewritten#404] [Suggestion Box]
If I'm viewing a post on a blog ([for example](https://inklesspen.tumblr.com/post/674101452176785408/j-cutter-thiefree-wrenhavenriver-why-must)), and that blog is one of the blogs I follow on my dash, then I can view the post on my dash by going to `tumblr.com/dashboard?max_post_id=[the post's id + 1]`. This is often handy for various hellsite reasons, like if I'm looking for an unsearchable post but I remember that it appeared on my dash near a particular post.

The URL transformation can be done manually, of course, but XKit 7 had the ["Go-to-Dash"](https://github.com/new-xkit/XKit/blob/master/Extensions/go_to_dash.js) extension that put a little icon in the upper right corner.

I'd like to port this to XKit Rewritten, but I think there's one glaring issue; XKit Rewritten doesn't run on `myblogname.tumblr.com`, just on the dash itself. Is there any chance this will change in the future?

![extension permissions in chrome](https://user-images.githubusercontent.com/11397/150653139-06014008-27fc-439e-bbb6-829904657dde.png)

**Comment by AprilSylph** [comment]
Nope, XKit Rewritten only targets parts of Tumblr that are React-based. If the page (or frame) isn't built on React, XKit 7 will continue to work there. Until all of Tumblr's web interface is updated to use React, XKit Rewritten should not be considered a 100% replacement for XKit 7.
